### ISSUE TITLE
Documentation issues

### ISSUE THREAD

**Issue by samnissen** [watir/watir#528]
There are some issues around the documentation for Watir:

### 1. How to get Firefox to work?

The Watir documentation > Firefox section says

> ## It Just Works™
> 
> Firefox is supported through a JavaScript driver, therefore ‘it just works™’ on all platforms.
> 
>     b = Watir::Browser.new :firefox

But I believe this is incorrect, as Firefox requires geckodriver to be installed. Can someone who knows how (i.e. not me) enumerate in the documentation how to get Watir to work with Firefox >= version 48? 

### 2. Documentation / documentation split

The 'Documentation' header links to http://watir.github.io/docs/home but the line on the homepage

> Read our [documentation](http://www.rubydoc.info/gems/watir-webdriver) to get started

links to http://www.rubydoc.info/gems/watir-webdriver . Which version is canonical? The rubydoc refers to Watir-Webdriver in places, so if the intended version, it probably needs an edit.

### 3. Blog missing

The link "More news ..." at the bottom of the home page links to http://watir.github.io/blog.html , which displays a Github 404 page. I believe it should link to http://watir.github.io/blog/ instead.

**Comment by mxygem** [comment]
Hey, @samnissen! 

Thanks for the input!

1. That's a good call here. I think the 'it just works' reference may be due to not needing to enable additional flags like you do with, say, Safari. I made [PR-98]()https://github.com/watir/watir.github.io/pull/98) for this but it's already addressed in another PR it looks like.
2. The ruby-docs line you referenced now points to the `watir` page instead of `watir-webdriver` so there's that. WRT which is canon: They both are, but they serve different purposes. The Watir documentation is meant for a higher level discussion, tutorials, examples, etc. The ruby-docs documentation would be for getting into the weeds of methods/classes. I've chatted with @titusfortner about this and we may need to do some more re-categorizing in general around this.
3. This is fixed now. It points to [http://watir.com/blog/](http://watir.com/blog/)
